src/app/dashboard/(create-event)/(steps)/general-info.tsx

```diff
@@ -306,6 +314,44 @@ export function GeneralInfoForm({
                 </FormItem>
               )}
             />
+            <FormField
+              name="termsLink"
+              control={form.control}
+              render={({ field }) => (
+                <FormItem className="col-span-2 flex flex-col gap-1">
+                  <FormLabel className="flex items-center gap-4">
+                    <span>Link do regulaminu</span>
+                    <Button
+                      asChild
+                      variant="eventGhost"
+                      size="sm"
+                      className="px-2 py-1"
+                    >
+                      <Link
+                        href="/regulamin-wydarzenia-dla-uczestnika-wzor.docx"
+                        download
+                        target="_blank"
+                      >
+                        <Download className="size-3" />
+                        Pobierz szablon regulaminu (współtworzony z Działem
```

Comment:
to trzeba poprawić na mobilce, myśle że można przycisk ogólnie dać pod polem to też będzie odstęp od etykiety taki sam jak w innych inputach 

<img width="593" height="210" alt="Image" src="https://github.com/user-attachments/assets/d34e44a0-0624-47e3-8bde-5ef7f77f283e" />